modules/33acrossBidAdapter.js

```diff
@@ -553,46 +636,66 @@ function _isIframe() {
 }
 
 // **************************** INTERPRET RESPONSE ******************************** //
-// NOTE: At this point, the response from 33exchange will only ever contain one bid
-// i.e. the highest bid
 function interpretResponse(serverResponse, bidRequest) {
-  const bidResponses = [];
-
-  // If there are bids, look at the first bid of the first seatbid (see NOTE above for assumption about ttx)
-  if (serverResponse.body.seatbid.length > 0 && serverResponse.body.seatbid[0].bid.length > 0) {
-    bidResponses.push(_createBidResponse(serverResponse.body));
-  }
+  let bidResponses = [];
+  const { seatbid, cur } = serverResponse.body;
+
+  if (!utils.isArray(seatbid)) {
+    return bidResponses;
+  }
+
+  // Pick seats with valid bids and convert them into an Array of responses
+  // in format expected by Prebid Core
+  bidResponses = (
+    seatbid
+      .filter((seat) => {
```

Comment:
I like the way you solved :) maybe the return statement is not entirely necessary, unless you want to make it more explicit.

e.g.
```
.filter((seat) => (utils.isArray(seat.bid) && seat.bid.length > 0))
```

also the `cur || USD` could be moved out of the loop, 
e.g.
```
  const { seatbid } = serverResponse.body;
  let { cur } = serverResponse.body;
  cur = cur || 'USD';
```

Reply:
I have removed the return statement. 

Also, I have decided to pass `cur` to the mapping callback rather than modifying the original bid object (a complete no! no!)